### DISCUSSION TITLE
Can no longer assign domains via the project page

### DISCUSSION THREAD

**Discussion by trm217** [vercel/community#2988] [Bug Report]
### Page to Investigate

Open the domain tab for a project

### Steps to Reproduce

When I try to add a new domain to a vercel project via the domain tab in the project page, the modal that opens after entering a domain crashes and instantly closes. Seems to be a new issue, worked fine a while back.
Anyways, I am no longer able to assign a domain at all.

**Comment by smaeda-ks** [comment]
@trm217 are you still seeing the issue? If so, could you open the DevTools and find the corresponding failing request and its error response (you may check console tab as well)?

**Reply by trm217** [reply]
Seems to be working again. thx
